custom_components/healthchecksio/sensor.py

```
@@ -0,0 +1,95 @@
+    DOMAIN_DATA,
+    ICON_DEFAULT,
+    ICON_DOWN,
+    ICON_GRACE,
+    ICON_PAUSED,
+    ICON_UP,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass, config_entry, async_add_devices):
```

Ruff (0.11.9)

Missing return type annotation for public function `async_setup_entry`

Add return type annotation: `None`

(ANN201)